website/cue/reference/components/sources/splunk_hec.cue

```diff
@@ -51,93 +52,8 @@ components: sources: splunk_hec: {
 		platform_name: null
 	}
 
-	configuration: {
-		acknowledgements: configuration._source_acknowledgements & {
-			type: object: {
-				options: {
-					max_number_of_ack_channels: {
-						common:      false
-						description: "The maximum number of Splunk HEC channels clients can use with this source. Minimum of `1`."
-						required:    false
-						type: uint: {
-							default: 1000000
-							unit:    null
-						}
-					}
-					max_pending_acks: {
-						common:      false
-						description: "The maximum number of ack statuses pending query across all channels. Equivalent to the `max_number_of_acked_requests_pending_query` Splunk HEC setting. Minimum of `1`."
-						required:    false
-						type: uint: {
-							default: 10000000
-							unit:    null
-						}
-					}
-					max_pending_acks_per_channel: {
-						common:      false
-						description: "The maximum number of ack statuses pending query for a single channel. Equivalent to the `max_number_of_acked_requests_pending_query_per_ack_channel` Splunk HEC setting. Minimum of `1`."
-						required:    false
-						type: uint: {
-							default: 1000000
-							unit:    null
-						}
-					}
-					ack_idle_cleanup: {
-						common:      false
-						description: "Whether or not to remove channels after idling for `max_idle_time` seconds. A channel is idling if it is not used for sending data or querying ack statuses."
-						required:    false
-						type: bool: {
-							default: false
-						}
-					}
-					max_idle_time: {
-						common:      false
-						description: "The amount of time a channel is allowed to idle before removal. Channels can potentially idle for longer than this setting but clients should not rely on such behavior. Minimum of `1`."
-						required:    false
-						type: uint: {
-							default: 300
-							unit:    "seconds"
-						}
-					}
-				}
-			}
-		}
-		address: {
-			common:      true
-			description: "The address to accept connections on."
-			required:    false
-			type: string: {
-				default: "0.0.0.0:\(_port)"
-			}
-		}
-		token: {
-			common:      true
-			description: "If supplied, incoming requests must supply this token in the `Authorization` header, just as a client would if it was communicating with the Splunk HEC endpoint directly. If _not_ supplied, the `Authorization` header will be ignored and requests will not be authenticated."
-			required:    false
-			warnings: ["This option has been deprecated, the `valid_tokens` option should be used."]
-			type: string: {
-				default: null
-				examples: ["A94A8FE5CCB19BA61C4C08"]
```

Comment:
Looks like this example wasn't carried over.

Reply:
> Looks like this example wasn't carried over.

Do we need examples for deprecated options?

Reply:
oh good point. I missed that. I don't think we do